### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by DanUshindi**
## Portfolio: details popup window

- You should implement the following interactions:
       When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
      When the user clicks (or taps) the close (X) button, the popup disappears.
      You should implement popups for both mobile and desktop screen sizes:
-

### CODE REVIEW

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @DanUshindi 😃, :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- ✔️The repo looks professional
- ✔️Git/GitHub flow best practices have been followed
- ✔️There are no linter errors
- ✔️You have a descriptive PR

## Required Changes ♻️

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -580,12 +586,187 @@ body {
...
+}
+
+.pop-image {
+  width: 364.88px;
+  height: 250.8px;
+  background: url("assets/images/project-one.png");
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+}
```
[the remark this answers is not in the document]
Comment (micheaol) [reply style.css:629]: Great work on making the changes requested by the previous reviewer but that has created another issue with your buttons. Kindly fix it. And please ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins).

Your Implementation | Figma Design
:---:|:---:
image | image

File: index.html [hunk 2]
```diff
@@ -18,7 +18,47 @@
...
+      <div class="pop-container">
+        <p class="about-popup"> A daily selection of privately
+          personalized reads; no accounts or sign-ups
+          required. has been the industry's standard
+          dummy text ever since the 1500s, when an
+          unknown printer took a standard dummy
+          text.</p>
+        <ul class="pop-tools-list  ">
+          <li class="tools-item right-border">
+            <a href="https://github.com/DanUshindi/Base/blob/main/style.css">css</a>
```
Comment (micheaol) [inline index.html:39]: Great work on making the changes requested by the previous reviewer but please ensure that all commented codes are removed from your codebase. It makes your code clean.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @DanUshindi , 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by prolajumokeoni — APPROVED** [review]
Hi @DanUshindi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


153888587-3e47b8be-12ea-47db-a3c2-33122e091533

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**